Modules/Filtering/LabelMap/include/itkShapeLabelObject.h

```diff
@@ -343,7 +330,7 @@ class ShapeLabelObject : public LabelObject<TLabel, VImageDimension>
   using OrientedBoundingBoxSizeType = Vector<double, VImageDimension>;
 
   using OrientedBoundingBoxVerticesType =
-    FixedArray<OrientedBoundingBoxPointType, IntegerPow<2, ImageDimension>::Result>;
+    FixedArray<OrientedBoundingBoxPointType, Math::UnsignedPower<unsigned int>(2, ImageDimension)>;
```

Comment:
Here it is, @blowekamp ensuring _at compile time_ that the power can be represented by an `unsigned int`, using the template argument that you requested 😃

Reply:
Great! Thank you for taking the time to improve the utility and generalization for your contribution.